app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -166,6 +166,12 @@ def perform
         create_statistical_matches
         generate_logging_info
 
+        if HmisExternalApis::AcHmis::Exporters::ClientExportUploader.can_run?
```

Comment:
Is this a good way to do this?

Reply:
Not sure if `HmisExternalApis::AcHmis::Exporters::ClientExportUploader.can_run?` is the right thing, but this is a fine place for it to live if it just needs to run once a day, and the timing isn't particularly important.

You'll want to wrap it in a "if driver loaded" statement too since this is the main app.

Reply:
Also, you may want to move this up to an earlier part of this file (see line 49 `sync_with_cas`) and throw it into a method something like `run_external_lookup_exports` or similar.  Since it should only take a few seconds to queue, that should be fine and will give a more consistent start time.